src/WCSimGenerator_Radioactivity.cc

```diff
@@ -199,64 +90,71 @@ void WCSimGenerator_Radioactivity::Initialize() {
 	
 	// Radon radioactivity constant
 	fRnLambda =  log(2) / (3.824*24.*3600.);
+	fRnLambda_Global = 0;
+		
+	// Parameter
+	CONC_BOTTOM 		= 2.63; // mBq/m^{3} -> From Nakano-san et al.
+	CONC_CENTER 		= 0.1;  // mBq/m^{3} -> From Nakano-san et al.
+	CONC_INTERMEDIATE 	= 0.3;  // Arbitrary
```

Comment:
Is there a good reason why these values are set here, rather than inside `RnModel_Fit_Params.hh`? Personally I like the new file (it means that all the calibration results (which will eventually come from a database, I presume) can be updated clearly independently from the actual code)

Reply:
Not really indeed, I moved them as pre-compiler constants inside RnModel_Fit_Params.hh